.gitattributes

```diff
@@ -0,0 +1 @@
+dist/** -diff linguist-generated=true 
```

Comment:
this can probably just be removed. it comes from the template i initially created the repo with